### PR TITLE
Add Rentable NFT ERC-4907

### PR DESCRIPTION
**Pull request by azoyan**

### CODE REVIEW

**Review by LouiseMedova — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
-version = "0.1.0"
-edition = "2021"
-license = "MIT"
+name = "rentable-nft"
+version = "0.2.7"
 authors = ["Gear Technologies"]
+edition = "2021"
 
 [dependencies]
 gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
```
Comment (bezrazli4n0) [inline Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: Cargo.toml [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
+primitive-types = { version = "0.12.1", default-features = false }
+nft-io = { path = "io" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
+gear-lib-derive = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
+sp-core-hashing = { version = "6.0.0", default-features = false }
+hashbrown = "0.13.1"
+gmeta = { git = "https://github.com/gear-tech/gear", branch = "stable" }
```
Comment (bezrazli4n0) [inline Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: Cargo.toml [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
 
 [dev-dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], branch = "stable" }
```
Comment (bezrazli4n0) [inline Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: Cargo.toml [hunk 4]
```diff
@@ -1,15 +1,39 @@
...
 gtest = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
```
Comment (bezrazli4n0) [inline Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: Cargo.toml [hunk 5]
```diff
@@ -1,15 +1,39 @@
...
+hex-literal = "0.3.4"
+sp-core = { git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
+gclient = { git = "https://github.com/gear-tech/gear.git", branch = "master" }
+tokio = { version = "1.23.0", features = ["full"] }
+blake2-rfc = { version = "0.2.18", default-features = false }
 
 [build-dependencies]
 gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
```
Comment (bezrazli4n0) [inline Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: state/Cargo.toml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+edition = "2021"
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear", branch = "stable" }
+gmeta = { git = "https://github.com/gear-tech/gear", branch = "stable", features = ["codegen"] }
+nft-io = { path = "../io" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
+
+[build-dependencies]
+gear-wasm-builder = { git = "https://github.com/gear-tech/gear", features = ["metawasm"], branch = "stable" }
```
Comment (bezrazli4n0) [inline state/Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: state/Cargo.toml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "nft-state"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear", branch = "stable" }
+gmeta = { git = "https://github.com/gear-tech/gear", branch = "stable", features = ["codegen"] }
```
Comment (bezrazli4n0) [inline state/Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: state/Cargo.toml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "nft-state"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear", branch = "stable" }
```
Comment (bezrazli4n0) [inline state/Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: io/Cargo.toml [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+license = "MIT"
+authors = ["Gear Technologies"]
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.4" }
+primitive-types = { version = "0.12.1", default-features = false }
+scale-info = { version = "2", default-features = false }
+parity-scale-codec = { version = "3", default-features = false }
+gmeta = { git = "https://github.com/gear-tech/gear.git", branch = "stable", features = ["codegen"] }
```
Comment (bezrazli4n0) [inline io/Cargo.toml]: Does it make sense to change branch to 'testnet'?

File: io/Cargo.toml [hunk 10]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "nft-io"
+version = "0.2.7"
+edition = "2021"
+license = "MIT"
+authors = ["Gear Technologies"]
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
```
Comment (bezrazli4n0) [inline io/Cargo.toml]: Does it make sense to change branch to 'testnet'?

**Review by bezrazli4n0 — APPROVED** [review]

**Review by fluiderson — APPROVED** [review]

**Pull request merged**